### ISSUE TITLE
1.6 corrupting NBT data

### ISSUE THREAD

**Issue by blablubbabcDEV** [Shopkeepers/Shopkeepers#38]
**Migrated from:** https://dev.bukkit.org/projects/shopkeepers/issues/28

**Originally posted by Darkhand81 (Aug 31, 2012):** 

Hiya Nisovin, v1.6 seems to have a serious issue with corrupting NBT data in save.yml. Luckily I keep backups! :)Bukkit build 2358.Â  After upgrading to 1.6 and restarting, Shopkeepers crashes with the following:http://hastebin.com/weganefoxu.avrasmAt the same time, the NBT data in save.yml is permanently corrupted.Before (from backup):http://hastebin.com/tanejufavu.vbsAfter:http://hastebin.com/lenerurinu.sm

**Comment by blablubbabcDEV** [comment]
**Originally commented by FlashIT (Aug 31, 2012):** 

Same problem?
After adding written books to our admin shopkeeper and reloading the plugin/server:
http://pastebin.com/K52PTftv

**Comment by blablubbabcDEV** [comment]
**Originally commented by Darkhand81 (Aug 31, 2012):** 

1.6.1 seems to have fixed it! Not sure if it's intended, but the binary line is still added to the file... the original text data is there as well though.
I did get duplicate regular villagers next to my shopkeepers the first time I ran 1.6.1 (with rebalance running), but I killed them off and restarted a few times, they haven't reappeared since.

            Edited Aug 31, 2012
